README.md

```diff
@@ -5,3 +5,11 @@ To set up a virtual environment for development, first [install pypoetry onto yo
 From there, you can [install your project dependencies](https://python-poetry.org/docs/basic-usage/#installing-dependencies) and
 [run any scripts in the library](https://python-poetry.org/docs/basic-usage/#using-poetry-run) within the virtualenv. You can also
 [build the contents of the repo into a python .whl](https://python-poetry.org/docs/cli#build) which can then be installed anywhere via `pip`
+
+
+### Manual Override
+Sometimes, the robots fail us and we have to take PDH into our own hands. Quick Commander_json manual update Guide.
+
+1) Make sure [Poetry](https://python-poetry.org/docs/) is installed on your local machine, this is necessary to run the scripts
```

Comment:
Not sure if its worth making this distinction but `poetry` technically isn't `necessary`, only python is necessary but poetry will coordinate installing all of your dependencies for you so you probably want it is the thing

Reply:
so I tried running with python directly w/o poetry and kept running into module issues, so I think we can either try to explain that in depth...or call poetry required

Reply:
What poetry does is create a python virtualenv for you and install dependencies into that. Then whenever you prefix a command with `poetry run ...` it essentially sources that virtualenv before running the rest of the command. So I'm not sure exactly what module errors you saw as a result of what commands but I guess regardless its best to just recommend that no one tries this without using poetry unless they really know what they're doing

Reply:
hmmm I did have a virtual env set up and all dependencies installed, but again must just be missing something and it comes down to how deep we wanna dig

Reply:
ran from the same environment I used to update the json manually with New Capenna. Module errors were on run update_json would fail at import statements saying it couldn't find scryfall_fetcher within the pdh_update_json module (or whatever the top level one is)

Reply:
I'll leave it up to you to determine how to instruct any other potential users on how to use this repo then but yeah I can guarantee you that poetry isn't doing anything that you couldn't do by hand, it just makes it easier to do those things